lib/SiftClient.php

```diff
@@ -16,6 +16,25 @@ class SiftClient {
     private $timeout;
     private $version;
 
+    /**
+     * @var \Psr\Log\LoggerInterface
+     */
+    private $logger;
+
+    /**
+     * @param \Psr\Log\LoggerInterface $logger
+     */
+    public function setLogger($logger)
```

Comment:
I'd suggest to add the signature here. `public function setLogger(LoggerInterface $logger)`